Workbooks/AKS/NPM Configuration/NPM Configuration.workbook

```diff
@@ -119,7 +119,7 @@
             "name": "clusterId",
             "type": 1,
             "isHiddenWhenLocked": true,
-            "value": "aks-e-workbook-update"
+            "value": ""
```

Comment:
fyi: removing the content of `value` doesnt remove a hardcoded value, it make the hardcoded value `""`.  if you want to remove the hardcoded value, remove this line entirely (and fix the preceeding line to not have a trailing comma)

no value = no value.  value = "" is a value that is the empty string, which may or may not work depending on the type of parameter, etc.